src/clob/clob.validators.ts

```diff
@@ -143,7 +147,7 @@ export const validatePostPerpOrderRequest: RequestValidator =
   );
 
 export const validateOrderRequest: RequestValidator = mkRequestValidator(
-  NETWORK_VALIDATIONS.concat([validateOrderId])
+  NETWORK_VALIDATIONS.concat([])
```

Comment:
This change seems unrelated to Kujira

Reply:
@fengtality

This was the simple change to enable the CLOB endpoints to an option to cancel all open orders at once.
But since our current Client version is not using this anymore, this is not needed anymore too.
I've removed it.